doc/glossary.rst

```diff
@@ -46,6 +46,20 @@ MNE-Python core terminology and general concepts
         :func:`mne.bem.make_watershed_bem` or :func:`mne.bem.make_flash_bem`.
         See :ref:`tut-forward` for usage demo.
 
+    dipole
+        See 'equivalent current dipole'.
+
+    equivalent current dipole
+        An equivalent current dipole (ECD) is an approximate representation of
+        post-synaptic activity in a small region of cortex. The intracellular
+        currents that give rise to measurable EEG/MEG signals are thought to
+        originate in populations of cortical pyramidal neurons aligned
+        perpendicularly to the cortical surface. Because the length of such
+        current sources is very small relative to the distance between the
+        cortex and the EEG/MEG sensors, the fields measured by the techniques
+        are well-approximated by, *i.e.*, equivalent to those generated by 
```

Comment:
confused by the `*i.e.*` here. It took me a few reads of this sentence for it to make sense. How about this:

```suggestion
        are well-approximated by (i.e., "equivalent" to) fields generated by 
```